client/signup/steps/woocommerce-install/transfer/index.tsx

```diff
@@ -99,12 +97,12 @@ export default function Transfer( { goToStep }: Props ): ReactElement | null {
 	const [ simulatedProgress, setSimulatedProgress ] = useState( 0.01 );
 	useEffect( () => {
 		const timeOutReference = setTimeout( () => {
-			if ( progress > simulatedProgress ) {
+			if ( progress > simulatedProgress || progress === 1 ) {
```

Comment:
sneaking in a bug fix where the transfer bar would swing between 1 and .95 instead of sit at .95 until completion is found